data/config_options.yml

```
@@ -21,6 +21,8 @@ options:
       type: bool
       default_value:
         markdown: "`false` / detected by system"
```

While we're at it, I think this needs changing as well: in the Node.js integration, the `active` state is not automatically set based on the presence of the push API key.

```suggestion
        markdown: "`false`"
```